# robot id
string id

# battery consumption
float32 battery_consumption

# time it took to complete the task
int32 time_for_completion

# distance travelled
float32 distance
